client/src/actions/habitActions.js

```diff
@@ -0,0 +1,67 @@
+import axios from 'axios';
+import { UPDATE_HABIT_LIST, ADD_HABIT, DELETE_HABIT, GET_ERRORS, UPDATE_HABIT } from './types';
+
+// Get the user's habits
+export const getHabits = () => dispatch =>
+	axios
+		.get('/api/habit/habits')
```

Comment:
I think that this should've to go to the user/habits route instead of the habit/habits. Missed this in the previous review it seems.

Reply:
Good idea -- that makes more sense. The request for that is still open so I'll work on it.

Reply:
Get habits pull request is currently open at #69 but it is still getting habits through the habit route -- @LydiaVuj is working on a user route now so perhaps this is something that can be revisited.

I'm changing this to point to `/api/user/habits` now, though.